### PR TITLE
[NEP-17984] add tooltip ability to header

### PR DESCRIPTION
**Pull request by vidishaweddy-readytech**
Add ability to add tooltip to header on table

### CODE REVIEW

File: example/views/index.haml [hunk 1]
```diff
@@ -229,9 +229,13 @@
                       %dt.heading{ data: { value: "Published On" } }
                       %dt.field{ data: { value: "published_on" } }
                       %dt.sortable{ data: { value: "published_on" } }
+                      %dt.tooltipContent{ data: { value: "<div style='text-align: left;'>This is tooltip<br /><b>High:</b> Test<div>".html_safe }}
```
Comment (hanpeic) [inline example/views/index.haml]: ideally the heading - data - value is HTML
and add one props:
```
%dt.heading-type{ data: { value: 'html'}}
```

Comment (vidishaweddy-readytech) [reply example/views/index.haml]: i make it the headingType to match the camel case we use on react

**Review by hanpeic — APPROVED** [review]
+1

**Comment by hanpeic** [comment]
+1

**Pull request merged**